optica.rb

```diff
@@ -14,7 +14,7 @@ class Optica < Sinatra::Base
   # endpoint for fab usage
   get '/roles' do
     # keys must be strings
-    return get_nodes(request, ['role', 'id', 'hostname'])
+    return get_nodes(request, ['role', 'id', 'hostname', 'roles', 'last_reported', 'environment'])
```

Comment:
Oof, yea. Maybe prefix with `_`, like `_fields`? In the future we might want to do stuff like `_count` too (`_count=1` is very useful)